contributor_roadmap.md

```diff
@@ -0,0 +1,165 @@
+# Contributor Roadmap/Expectations
+Hello, thank you for your interest in contributing to Matplotlib! 
+
+Matplotlib is primarily administered via github, so more information on the exact permissions granted to each role is available at [github help](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level)
+
+
+# Github
+
+* https://github.com/matplotlib/matplotlib
+
+## Github Organization ownership
+Full control of everything on GH
+**People:**
+- Lead developer, steering council
+- someone from NF (?)
+- someone from outside?
```

Comment:
What would the criteria be for this "outsider" to have access? 
Mainly as to define the role and reach of this person tasks